MAUI/TimePicker/overview.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: About .NET MAUI Time Picker control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Time Picker (SfTimePicker) control, its basic features, and time picker functionalities.
+platform: maui
+control: SfTimePicker
+documentation: ug
+---
+
+# Overview of .NET MAUI Time Picker (SfTimePicker)
+
+Syncfusion .NET MAUI Time Picker (SfTimePicker) control allows you to select time and visualizes its items inside a popup or a drop-down UI element. The Time Picker also enables you to set time formats, and to customize its dialog appearance by configuring its header, footer, and more.
+
+## Key features
+
+* **Header view**: The Time Picker allows you to add and customize header text.
+
+* **Column header view**: The .NET MAUI Time Picker supports a customizable column header view to separate headings for each column.
+
+* **Footer view**: Provides validation buttons (OK and Cancel) in the footer view. The footer text and background color can be customized.
+
+* **Selection view**: Selection view is used to show the selected time and it can be customized.
+
+* **Time format**: Offers 9 predefined formats to represent the value of the time.
+
+* **Picker mode**: Show the picker in a Popup with a dialog mode. The relative dialog mode is used to align the picker in a specific position.
+
+* **Intervals**: The time values can be populated with intervals for hours, minutes, and seconds, individually.
```

Comment:
* **Intervals**: The time values can be populated individually with intervals for hours, minutes, and seconds.

Reply:
Content modified.